packages/rest/src/__tests__/acceptance/validation/validation.acceptance.ts

```diff
@@ -93,6 +104,57 @@ describe('Validation at REST level', () => {
     });
   });
 
+  context('with request body validation options', () => {
+    class ProductController {
+      @post('/products')
+      async create(
+        @requestBody({required: true}) data: Product,
+      ): Promise<Product> {
+        return new Product(data);
+      }
+    }
+
+    before(() =>
+      givenAnAppAndAClient(ProductController, {
+        nullable: false,
+        skipCache: true,
+      }),
+    );
+    after(() => app.stop());
+
+    it('rejects requests nullable option is false', async () => {
```

Comment:
"(it) rejects requests nullable option is false" does not reads like an English sentence to me. Did you mean the following?

```suggestion
    it('rejects requests when nullable option is false', async () => {
```